.auditrc.json

```diff
@@ -3,5 +3,5 @@
     "788",
     "813"
   ],
-  "verified_hash": "730da374ff70078b87ed3899d46614a8"
+  "verified_hash": "e6935c48f5c7b1cc62dfe3ba13a181c1"
```

Comment:
Curious why this is required?

Reply:
It is related to a change that @garethbowen implemented (we discussed it yesterday during standup). It is not clear to me why (and Gareth is investigating) but there was a mismatch between the hash of `package-lock.json` and the `verified_hash` hash in `.auditrc.json`. It is puzzling because as you can notice, I did not change `package-lock.json`.